test/e2e/metering_manual_install_tests.go

```diff
@@ -107,6 +112,105 @@ func testManualMeteringInstall(
 	assert.NoError(t, err, "capturing logs and uninstalling metering should produce no error")
 }
 
+func createNFSProvisioner(ctx *deployframework.DeployerCtx) error {
+	//Waiting for Metering pods
+	ctx.TargetPodsCount = nonHDFSTargetPodCount
+
+	_, err := createTestingNamespace(ctx.Client, "nfs")
+	if err != nil {
+		return err
+	}
+
+	files := map[string]string{
+		"pod":          "/test/testhelpers/nfs_server.yaml",
+		"service":      "/test/testhelpers/nfs_service.yaml",
+		"storageClass": "/test/testhelpers/nfs_storageclass.yaml",
+
+	}
+	server := &corev1.Pod{}
+	service := &corev1.Service{}
+	storageClass := &v1beta1.StorageClass{}
```

Comment:
After renaming the `k8s.io/api/storage/v1beta1` to `storagev1 "k8s.io/api/storage/v1`, use the following:
```suggestion
	storageClass := &storagev1.StorageClass{}`
```